types/smtp-server/index.d.ts

```diff
@@ -1,9 +1,12 @@
 // Type definitions for smtp-server v1.15.0
 // Project: https://github.com/andris9/smtp-server
 // Definitions by: markisme <https://github.com/markisme>
+//                 taisiias <https://github.com/Taisiias>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped 
 
-export class SMTPServer {
+/// <reference types="node"/>
+
```

Comment:
Maybe we can `import { EventEmitter } from 'events';` here, And then just `export class SMTPServer extends EventEmitter...`. What do you think?

Reply:
That's OK with me. Done.